extension/src/paymentHandler/payment-handler.js

```diff
@@ -37,6 +40,21 @@ async function handlePayment(paymentObject) {
       data: paymentValidator.buildCtpErrorResponse(),
     }
 
+  if (!auth.isAuthorized(paymentObject, authToken)) {
+    console.log('unauthorized ...')
```

Comment:
Remove this console.log

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/b9321da71d385fd3351ad010dcb81ea5a0a0f52c